city_scrapers/spiders/chi_northeastern_il_university.py

```
@@ -0,0 +1,126 @@
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def getMeetingDetails(self, response):
+        print(response.text)
+
+    def _parse_title(self, item):
+        return item if not item == "" else "BOARD MEETING"
```

Ruff

Use `item != ""` instead of `not item == ""`

Replace with `!=` operator

(SIM201)